src/state/schedule-data/foundation-info/foundation-info.model.ts

```diff
@@ -31,6 +31,14 @@ export interface FoundationInfoModel {
   dates: number[];
 }
 
+export interface ServerInfoModel {
+  children_number?: number[];
+  extra_workers?: number[];
+  frozen_shifts?: [number | string, number][];
+  dates: number[];
+  holidays: number[];
+}
```

Comment:
Proponuję nazwą `ServerFoundationInfoModel`, bo `ServerInfoModel` nasuwa myśl, że to zawiera informację o serwerze